support-frontend/assets/helpers/subscriptionsForms/formValidation.js

```diff
@@ -59,12 +59,12 @@ function withDeliveryValidation(state: WithDeliveryCheckoutState): AnyErrorType[
       errorAction: setFormErrors,
     }: Error<FormField>),
     ({
-      errors: applyAddressRules(getDeliveryAddressFields(state)),
+      errors: applyAddressRules(getDeliveryAddressFields(state), 'delivery'),
       errorAction: setAddressFormErrorsFor('delivery'),
     }: Error<AddressFormField>),
     ...(shouldValidateBillingAddress(formFields) ? [
       ({
-        errors: applyAddressRules(getBillingAddressFields(state)),
+        errors: applyAddressRules(getBillingAddressFields(state), 'billing'),
```

Comment:
ah this is an interesting change - it intersects with Damien's PR for HD because the postcode check only happens in `'delivery'` address rules.  It'll be his conflict to merge probably though 😄